.repo-metadata.json

```diff
@@ -2,11 +2,12 @@
     "name": "google-cloud-core",
     "name_pretty": "Google API client core library",
     "client_documentation": "https://cloud.google.com/python/docs/reference/google-cloud-core/latest",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "CORE",
     "repo": "googleapis/python-cloud-core",
     "distribution_name": "google-cloud-core",
     "default_version": "",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "google-cloud-core"
```

Comment:
This library is not a GAPIC*, is an `api_shortname` required?

Reply:
Fixed.